Mecrano/Personal-Page
# Refactor/smart and dumb components concept

#### What problem is this solving?

Improve code read and performance

#### How to test it?

1. Run `yarn dev`
2. See changes

#### Screenshots or example usage:

![image](https://user-images.githubusercontent.com/26680887/187518556-0ec4b0a9-cc48-4056-a088-372fd504d5dd.png)

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/RbDKaczqWovIugyJmW/giphy.gif)
